src/app/services/technical-record/technical-record.service.spec.ts

```diff
@@ -216,4 +217,218 @@ describe('TechnicalRecordService', () => {
       expect(service.getMakeAndModel({ techRecord_make: undefined, techRecord_model: undefined } as V3TechRecordModel)).toBe('');
     });
   });
+
+  describe('haveAxlesChanged', () => {
+    it('should return true if a property of the hgv gross axle has changed', () => {
```

Comment:
You could probably parameterise these tests

Reply:
We had a look at that but we thought this was more readable